packages/react-instantsearch-hooks-dom/src/widgets/Hits.tsx

```diff
@@ -13,7 +13,9 @@ export type HitsProps<THit extends BaseHit> = Omit<
 > &
   UseHitsProps<THit>;
 
-export function Hits<THit extends BaseHit = BaseHit>(props: HitsProps<THit>) {
+export function Hits<THit extends BaseHit = BaseHit>({
+  ...props
```

Comment:
Why this change?

Reply:
needed it for classnames at widget level at some point, no longer now so can indeed be reverted